tetris.sh

```diff
@@ -17,13 +17,60 @@
 #
 # Author: Kirill Timofeev <kt97679@gmail.com>
 #
+# Localized Support: Rojen Zaman <rojen@riseup.net> | lang/README.md
+#
 # This program is free software. It comes without any warranty, to the extent
 # permitted by applicable law. You can redistribute it and/or modify it under
 # the terms of the Do What The Fuck You Want To Public License, Version 2, as
 # published by Sam Hocevar. See http://www.wtfpl.net/ for more details.
 
 set -u # non initialized variable is an error
 
+# show help page when given -h argument
+function help_page() {
+cat <<EOF
+usage: ./`basename $0` -l <language>
+supported languages:
+english (it is defualt, option not required)
+$(ls -I README.md -1t lang/ | cut -d'.' -f1)    
```

Comment:
Let's imagine that somebody copied the script and there is no lang directory at all. How script should work in this situation?

Reply:
we will be control the lang dir

Reply:
I'm sorry, what do you mean?

Reply:
for this
> Let's imagine that somebody copied the script and there is no lang directory at all. How script should work in this situation?

I will add code for check the `lang/` diroctery